### PR TITLE
Update CCIE docs index page with system requirements, options, cases studies, and links

### PR DESCRIPTION
**Pull request by michelle-luna**

### CODE REVIEW

File: jekyll/_ccie/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+Welcome to Enterprise Documentation for CircleCI! The CircleCI Enterprise solution is installable inside your private cloud or data center and is available to try with a free license. If you are new to CircleCI Enterprise administration, check out the [Try CircleCI Enterprise]({{ site.baseurl }}/enterprise/single-box/) document for trial installation instructions and links to the machine images. Then, see the [Overview]({{ site.baseurl }}/enterprise/overview/) document for details about the features, build environments, and architecture. If you are a developer, refer to [Setting Up Projects]({{ site.baseurl }}/enterprise/quick-start) for information about adding and debugging projects.
+
+## Hardware Platform Support and Requirements
+
+- Minimum of two machines or instances, one for the Services component and one for the Builders 
+- The Services machine must have at least four CPUs or GPUs, 8GB RAM, and 30GB of disk space 
+- The Builders machine must have six CPUs or GPUs and 10GB RAM and a root volume of at least 50GB
+- Both machines must have network access to a GitHub Enterprise instance
```
Comment (drazisil) [inline jekyll/_ccie/index.md]: CCIE works with normal GitHub, not just GHE

File: jekyll/_ccie/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+title: "Enterprise Documentation"
+description: "Landing page for Enterprise CircleCI Documentation"
+permalink: /enterprise/
+---
+
+Welcome to Enterprise Documentation for CircleCI! The CircleCI Enterprise solution is installable inside your private cloud or data center and is available to try with a free license. If you are new to CircleCI Enterprise administration, check out the [Try CircleCI Enterprise]({{ site.baseurl }}/enterprise/single-box/) document for trial installation instructions and links to the machine images. Then, see the [Overview]({{ site.baseurl }}/enterprise/overview/) document for details about the features, build environments, and architecture. If you are a developer, refer to [Setting Up Projects]({{ site.baseurl }}/enterprise/quick-start) for information about adding and debugging projects.
+
+## Hardware Platform Support and Requirements
+
+- Minimum of two machines or instances, one for the Services component and one for the Builders
```
Comment (drazisil) [inline jekyll/_ccie/index.md]: Are we depreciating the single-box install option?

File: jekyll/_ccie/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## Hardware Platform Support and Requirements
+
+- Minimum of two machines or instances, one for the Services component and one for the Builders 
+- The Services machine must have at least four CPUs or GPUs, 8GB RAM, and 30GB of disk space 
+- The Builders machine must have six CPUs or GPUs and 10GB RAM and a root volume of at least 50GB
+- Both machines must have network access to a GitHub Enterprise instance
+- Amazon AWS using the CircleCI AMI 
+
+OR
```
Comment (drazisil) [inline jekyll/_ccie/index.md]: This part is confusing to me. It seems like you need all of the above, OR AWS.

File: jekyll/_ccie/index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- Private cloud running a Linux distribution with Docker support, such as Ubuntu Trusty 
+
+To determine the size required for your workload, provision two CPUs for processing plus two CPUs and 4GB of memory for each container. For example, six CPUs and 10GB of RAM supports two containers concurrently because the default container size is two CPUs with 4GB. 
+
+## Installation Options
+
+It is possible to install the Enterprise CircleCI solution in either of the following configurations:
+
+- A single Services machine with one or more Builder machines.
+- Two Services machines for HA with one or more Builder machines, see [High Availability]({{ site.baseurl }}/enterprise/single-box/) for instructions.
```
Comment (drazisil) [inline jekyll/_ccie/index.md]: Single-box install is just that, a single box, not a services box and a builder box. Possible wrong link?

File: jekyll/_ccie/index.md (hunk 1 above)
Comment (michelle-luna) [inline jekyll/_ccie/index.md]: thanks, fixed.

File: jekyll/_ccie/index.md (hunk 2 above)
Comment (michelle-luna) [inline jekyll/_ccie/index.md]: no, I've added this

File: jekyll/_ccie/index.md (hunk 3 above)
Comment (michelle-luna) [inline jekyll/_ccie/index.md]: thanks, I reworked this to make it clear

File: jekyll/_ccie/index.md (hunk 4 above)
Comment (michelle-luna) [inline jekyll/_ccie/index.md]: good catch, fixed

**Pull request merged**